www/docs/actions.mdx

```diff
@@ -0,0 +1,287 @@
+---
+id: actions
+title: Actions and Suspensions
```

Comment:
Maybe `Suspect and Actions` to be in symmetry with the order in the article?

Reply:
Should this be 2 separate pages?

Reply:
There’s a bit of a paradox because suspend is a little light and doesn’t really make sense outside of the context of other things, but you also kind of need to know how it works to understand actions…

I can have a play with it.